### PR TITLE
[de] added gGEC AP

### PR DESCRIPTION
**Pull request by Luisa-LT**
https://github.com/languagetooler-gmbh/languagetool-premium/issues/6692

### CODE REVIEW

**Comment by tiff** [comment]
Looks good to me. In a subsequent PR please also look into "deinetwegen", "euretwegen", "ihrerseits", "deinerseits", ...

File: languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/remote-rule-filters.xml [hunk 1]
```diff
@@ -1959,7 +1959,7 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
         <rulegroup id="AI_DE_GGEC_REPLACEMENT_ORTHOGRAPHY_UPPERCASE" name="">
             <rule><!--1-->
                 <pattern case_sensitive="yes">
-                    <token regexp="yes">Du|Dir|Dich|Dein(e[rnms]?)?|Euch|Eue?re[rmns]?|Ihr(e(mnrs)?|s)?|Ihnen|Ihretwegen|Deinetwegen|Sie</token>
+                    <token regexp="yes">Du|Dir|Dich|Dein(e[rnms]?)?|Euch|Eue?re[rmns]?|Ihr(e(mnrs)?|s)?|Ihnen|Ihretwegen|Euretwegen|Deinetwegen|Sie|Ihrerseits|Deinerseits|Eurerseits</token>
```
Comment (tiff) [inline languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/remote-rule-filters.xml:1962]: Auch "Euerseits" (ohne das "r") aber ich merge erstmal, damit das morgen live ist

Comment (Luisa-LT) [reply languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/remote-rule-filters.xml:1962]: Thanks!

**Comment by danielnaber** [comment]
More words: deinesgleichen, deinethalben, deinetwillen etc.

**Comment by Luisa-LT** [comment]
https://github.com/languagetool-org/languagetool/pull/9654/commits/8f5b05b5643ab9499d610cbf33d1dd8526d40371

**Pull request merged**